adam/build/Dockerfile

```diff
@@ -1,19 +1,7 @@
-FROM ubuntu
+FROM computationalgenomicslab/spark-and-maven:1.5.2.3.3.3-build
```

Comment:
Don't really like the image name ```spark-and-maven```, but not sure I could think of anything better.